filetracker/migration_test.py

```diff
@@ -0,0 +1,111 @@
+"""Integration tests for client-server interaction."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+import signal
+import tempfile
+import time
+import unittest
+from wsgiref.simple_server import make_server
+
+from filetracker.client import Client, FiletrackerError
+from filetracker.servers.migration import MigrationFileTrackerServer
+from filetracker.servers.files import FileTrackerServer
+
+_TEST_PORT_NUMBER = 45735
+
+
+class InteractionTest(unittest.TestCase):
```

Comment:
Shouldn't it be `MigrationTest`?

Reply:
Of course, I was in a rush, should have copy-pasted more carefully. Will fix in a moment.